docs/src/main/sphinx/release/release-434.md

```diff
@@ -0,0 +1,102 @@
+# Release 434 (17 Nov 2023)
+
+## General
+
+* {{breaking}} Rename the `query.max-writer-tasks-count` configuration property
+  and the related session property `max_writer_tasks_count` to
+  `query.max-writer-task-count` and `max_writer_task_count`. ({issue}`19793`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+## Accumulo connector
+
+## Atop connector
+
+## BigQuery connector
+
+* Support reading `json` type. ({issue}`19790`)
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+* Improve performance when reading large checkpoint files on partitioned tables.
+  ({issue}`19588`)
+
+## Druid connector
+
+## Elasticsearch connector
+
+## Google Sheets connector
+
+## Hive connector
+
+* Add timestamp to date schema evolution. ({issue}`19513`)
+* Add tracing to metastore access. ({issue}`19712`)
+
+## Hudi connector
+
+## Iceberg connector
+
+* {{breaking}} Remove support for legacy table statistics tracking. ({issue}`19803`)
+
+## Ignite connector
+
+## JMX connector
+
+## Kafka connector
+
+## Kinesis connector
+
+## Kudu connector
+
+## Local file connector
+
+## MariaDB connector
+
+## Memory connector
+
+## MongoDB connector
+
+## MySQL connector
+
+## Oracle connector
+
+## Phoenix connector
+
+## Pinot connector
+
+## PostgreSQL connector
+
+## Prometheus connector
+
+## Raptor connector
+
+## Redis connector
+
+## Redshift connector
+
+## SingleStore connector
+
+## SQL Server connector
+
+## TPC-H connector
+
+## SPI
+
+* Add bulk append methods to `BlockBuilder`. ({issue}`19577`)
+* {{breaking}} Remove `VariableWidthBlockBuilder` `buildEntry` method. ({issue}`19577`)
```

Comment:
@dain 

or should we use 

```
VariableWidthBlockBuilder#buildEntry
```

Reply:
I'd normally do `VariableWidthBlockBuilder#buildEntry`

Reply:
seems like we used dot in the past..